openqabot/types/aggregate.py

```diff
@@ -95,8 +95,16 @@ def __call__(
                     # if filtering embargoed updates is off we ignoring this field
                     else:
                         valid_incidents.append(i)
+
+            # filter out incidents which have already failed in single incident tests
+            filtered_incidents = [
+                incident
+                for incident in valid_incidents
+                if not incident.has_failures(token)
+            ]
+
```

Comment:
```suggestion
            filtered_incidents = [i for i in valid_incidents if not i.has_failures(token)]
            log.info("filtered out incidents with failed single incident tests: %s", ",".join(i for i in filtered_incidents))
```

This way we don't need the comment but instead have an informative message

Reply:
Nope, I prefer to leave the presence/absence of messages to has_failures

Reply:
That does not resolve my point though. Then log in has_failures and leave the the comment for good.

Reply:
That's whats gonna happen